specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -1146,12 +1146,12 @@
       "properties": {
         "name": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
           "description": "Name of the configuration."
         },
         "version": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
May I ask what's the different with here "[version](https://github.com/Azure/azure-rest-api-specs/blob/4bab0bfdf7b1e2e3a50eeda420c2891d087d1d85/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L801)".